recipes/graphql-doc

```diff
@@ -0,0 +1,4 @@
+(graphql-doc
+	:fetcher github
+	:repo "ifitzpatrick/graphql-doc.el"
+	:files ("graphql-doc.el"))
```

Comment:
Prefer the default recipe if possible -- this appears to be equivalent: `(graphql-doc :repo "ifitzpatrick/graphql-doc.el" :fetcher github)`